storage/tree_validation_test.go

```diff
@@ -87,6 +87,94 @@ func TestValidateTreeForCreation(t *testing.T) {
 	}
 }
 
+func TestValidateTreeForUpdate(t *testing.T) {
+	tests := []struct {
+		desc     string
+		updatefn func(*trillian.Tree)
+		wantErr  bool
+	}{
+		{
+			desc: "valid",
```

Comment:
I'd have expected the values for the `desc` and `wantErr` fields to line up, but it looks like having the `func`s in there stops `gofmt` doing that.  Odd, but as-is is fine.